sdcm/utils/nemesis_utils/indexes.py

```diff
@@ -0,0 +1,101 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2023 ScyllaDB
+
+import logging
+import random
+import time
+from typing import Tuple
+
+from cassandra.query import SimpleStatement  # pylint: disable=no-name-in-module
+
+from sdcm.cluster import BaseNode
+from sdcm.sct_events import Severity
+from sdcm.sct_events.database import DatabaseLogEvent
+from sdcm.sct_events.filters import DbEventsFilter
+from sdcm.sct_events.system import InfoEvent
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_table_with_biggest_partitions_count(session) -> Tuple[str, str]:
```

Comment:
It is not clear why using this method. Why limiting the tables to be selected for creating SI?
IIUC, a test can have, for example, a s-b table with large-partitions, but it will never be tested if there's a c-s table with a higher number of partitions. Why not give a chance to all user tables to be tested?

Reply:
You seem to be right. I was thinking that there are small ks/cf in the cluster, but apparently there are not many. I switched it to `get_non_system_ks_cf_list` with MV exclude.